lib/kitchen/oven.rb

```diff
@@ -12,10 +12,16 @@ class Oven
     # @param recipes [Array<Recipe>] an array of recipes with which to bake the document
     # @param output_file [String] the path to the output file
     #
-    def self.bake(input_file:, recipes:, output_file:, config_file: nil)
+    def self.bake(input_file:, recipes:, output_file:, config_file: nil, locales: nil)
       profile = BakeProfile.new
       profile.started!
 
+      if locales
+        I18n.load_path << file_glob("#{__dir__}/#{locales}/*.yml")
+        I18n.backend.reload!
+        # TODO: reload! clears the symbols that are manually stored from TRANSLITERATIONS in openstax_kitchen.rb. Fix: add these to the load_path instead?
```

Comment:
@jpslav Concerns I have: 1) the comment, 2) currently in recipes, I am able to pass a locales path that references the locally mounted copy of kitchen. Will this work when not using a local version of kitchen?

Reply:
See my PR into your PR.  It is a different approach that avoids modifying the Kitchen I18n backend.  Also in my PR you'll see that it auto-detects a `locales` directory next to the recipe or you can pass in a path to another locales folder.  The trick when passing in a path is to use `__dir__` to reference the current file's directory.

Reply:
See my PR into your PR.  It is a different approach that avoids modifying the Kitchen I18n backend.  Also in my PR you'll see that it auto-detects a `locales` directory next to the recipe or you can pass in a path to another locales folder.  The trick when passing in a path is to use `__dir__` to reference the current file's directory.